### PR TITLE
Heading Component

### PR DESCRIPTION
**Pull request by codeswan**
A Heading component that allows users to select from different heading levels (`h1` through `h6`), as well as specify whether the heading should be bold or not.

### CODE REVIEW

File: src/scss/grommet-core/_objects.heading.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.heading {
+  margin-bottom: halve($inuit-base-spacing-unit);
+
+  &--large {
+    font-size: 60px;
```
Comment (ericsoderberghp) [inline src/scss/grommet-core/_objects.heading.scss]: I would think the large and small would be relative to the tag chosen. Is that not the case, that setting large or small fixes the font size irrespective of the tag?

File: src/scss/grommet-core/_objects.heading.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.heading {
+  margin-bottom: halve($inuit-base-spacing-unit);
+
+  &--large span {
+    font-size: 125%;
+    line-height: 60px;
```
Comment (ericsoderberghp) [inline src/scss/grommet-core/_objects.heading.scss]: I think the line-height will need to be adjusted relatively as well. We won't a 60px line-height for a small h5, for example. I'd set it to 1.2 for now and we can adjust when we tune the sizes.

Comment (codeswan) [inline src/scss/grommet-core/_objects.heading.scss]: Updated.

**Comment by ericsoderberghp** [comment]
Thanks. Good for now. We can tweak later.

**Pull request merged**